### PR TITLE
styling started for the recipe card component

### PR DESCRIPTION
**Pull request by gabrielaehrenbrink**
Created a bit more of the recipe card component, might need to adjust styling once the actual data is coming in

### CODE REVIEW

**Comment by gabrielaehrenbrink** [comment]
for the my recipes page we need to decide if we want   const response = await fetch(`${BACKEND_URL}/recipes`, requestOptions); or  do we change it to match the routes on the backend to  /myrecipes/:user_id ?

**Comment by kawrou** [comment]
> for the my recipes page we need to decide if we want const response = await fetch(`${BACKEND_URL}/recipes`, requestOptions); or do we change it to match the routes on the backend to /myrecipes/:user_id ?

I thought I changed it. I think it needs to change otherwise it won't use the correct route.

**Comment by gabrielaehrenbrink** [comment]
Yeah I changed it just so it would work but at the moment, using /recipes but I think myrecipes/user_id would be best

File: api/controllers/recipes.js [hunk 1]
```diff
@@ -215,16 +215,22 @@ const getRecipeById = async (req, res) => {
 };
 
 const getAllRecipesByUserId = async (req, res) => {
-  const userId = req.params.user_id;
-  const token = generateToken(req.user_id);
   try {
+    const user = await User.findOne({ _id: req.user_id });
```
Comment (kawrou) [inline api/controllers/recipes.js:223]: Good catch. Didn't think about that.

File: frontend/src/components/Recipe/RecipeCard.jsx [hunk 2]
```diff
@@ -1,5 +1,70 @@
...
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
+  const [recipeData, setRecipeData] = useState(null);  
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const token = localStorage.getItem("token");
+        const data = await getRecipeById(recipe._id, token);
```
Comment (kawrou) [inline frontend/src/components/Recipe/RecipeCard.jsx:20]: Is this for getting recipe data for individual recipes?

**Pull request closed without merging**